live-examples/js-examples/array-sort.html

```diff
@@ -0,0 +1,16 @@
+<pre>
+<code id="static-js">var array1 = ['March', 'Jan', 'Feb', 'Dev'];
```

Comment:
I guess "Dev" should be "Dec". Remember to change the "expected output" line as well.

Reply:
Also if these are months, I think we could use the descriptive variable name "months" instead of "array1".

Reply:
it was a typo, I fixed it.  :smiley: